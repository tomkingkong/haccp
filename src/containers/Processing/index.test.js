import React from 'react';
import { shallow } from 'enzyme';

import { Processing } from '.';

describe('Processing', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Processing />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});