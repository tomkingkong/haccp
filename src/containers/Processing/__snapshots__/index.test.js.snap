// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Processing should match snapshot 1`] = `
<div>
  <h2>
    Processing
  </h2>
  <button
    onClick={[Function]}
  >
    Next Page
  </button>
</div>
`;
